zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/Zab1_0Test.java

```diff
@@ -69,15 +69,14 @@
 import org.apache.zookeeper.txn.TxnHeader;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class Zab1_0Test extends ZKTestCase {
 
     private static final Logger LOG = LoggerFactory.getLogger(Zab1_0Test.class);
 
-    private static final File testData = new File(System.getProperty("test.data.dir", "src/test/resources/data"));
```

Comment:
is this correct ?
is the test trying to read some existing file generated with an old version of ZK ?

Reply:
previously, the tests in this file were creating tempDirectories under zookeeper-server/src/test/resources/data and the createQuorumPeer would use that directory for eg:

zookeeper-server/src/test/resources/data/test.dir/version-2
And once the tests finish running the test.dir and version-2 directories gets deleted.

With this change, the tests would create tempDirectories and files under the system temp

/tmp/uniqid/test.dir/version-2